lib/headings.js

```diff
@@ -1,37 +1,68 @@
-var fmt = require('util').format
+var cheerio = require('cheerio')
```

Comment:
This `headings.js` file is now a plugin for `markdown-it` rather than a standalone Node module for the rest of marky to consume (the only substantial effect of this change is to move the one line where the module is used from `index.js` to `render.js`).

@ashleygwilliams do you think it's worth splitting into a completely separate package and publishing it on NPM? It's pretty specific to marky, since it's basically "a markdown-it plugin for generating GitHub-like headings for applications that *also* use the markdown-it-emoji plugin"